### ISSUE TITLE
🐛 camera freezing when pointing to a "Face"

### ISSUE THREAD

**Issue by marcosrjjunior** [cuvent/react-native-vision-camera#3055]
### What's happening?

The camera is freezing when pointing to a face. Please check videos below.

[30 fps](https://www.youtube.com/watch?v=RHcQgXXmaCc)
[60 fps](https://youtu.be/JDwVI1lMBNg)

Found these issues pointing to the same error: https://github.com/mrousavy/react-native-vision-camera/issues/2313, https://github.com/mrousavy/react-native-vision-camera/pull/2329. Not sure if is direct related because I'm not even using any zoom option.

Seems like the camera is trying to perform some face identification when focus is on a face. I'm guessing it is related to the iOS version.

**
- This issue is also happening when using a different camera library.
- I could not reproduce on the "ShadowLens 1.1.2 (17)" app.

### Reproduceable Code

```tsx
function App() {
  const device = useCameraDevice('back')

  if (device == null) return <View />

  return (
    <Camera
      style={StyleSheet.absoluteFill}
      device={device}
      isActive
      enableZoomGesture={false}
    />
  )
}
```

### Relevant log output

```shell
output on the videos above.

[unknown/unknown: Error Domain=AVFoundationErrorDomain Code=-11819 "Cannot Complete Action" UserInfo={NSLocalizedDescription=Cannot Complete Action, NSLocalizedRecoverySuggestion=Try again later.}]
```
![IMG_0165](https://github.com/mrousavy/react-native-vision-camera/assets/5287262/6fcac147-b3fc-44f0-bf13-6bb2038734d9)
![IMG_0163](https://github.com/mrousavy/react-native-vision-camera/assets/5287262/334a938b-8d44-43a8-a61f-94f861f62840)

### Camera Device

```json
{
  "isMultiCam": false,
  "minFocusDistance": 10,
  "formats": [],
  "name": "Back Camera",
  "minExposure": -8,
  "hasFlash": false,
  "id": "com.apple.avfoundation.avcapturedevice.built-in_video:0",
  "physicalDevices": [
    "wide-angle-camera"
  ],
  "neutralZoom": 1,
  "supportsFocus": true,
  "supportsRawCapture": false,
  "hasTorch": false,
  "maxZoom": 95.625,
  "maxExposure": 8,
  "supportsLowLightBoost": false,
  "sensorOrientation": "landscape-left",
  "minZoom": 1,
  "hardwareLevel": "full",
  "position": "back"
}
```

### Device

iPad iOS 17.5.1 9th gen.
Couldn't reproduce on 8th gen or iPhone 13 Pro.
Maybe something related to the hardware performance.

### VisionCamera Version

first identified on "2.16.7", also happening on 4.3.2 after update

### Can you reproduce this issue in the VisionCamera Example app?

I didn't try but followed examples and even tried disabling some props.

### Additional information

- [X] I am using Expo
- [ ] I have enabled Frame Processors (react-native-worklets-core)
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by arjunmenon1302** [comment]
bump

**Comment by mrousavy** [comment]
@arjunmenon1302 why are you bumping an issue on the same exact day it was created at?

**Comment by mrousavy** [comment]
@marcosrjjunior sorry I cannot help you. I cannot reproduce the issue with the sample code you provided (tested on iPhone 15 Pro and iPhone 8), and if you didn't try to reproduce it in the example app then we don't know where it is coming from. Maybe you are using a Frame Processor Plugin that does something weird.

**Comment by kurt-idu** [comment]
bump

**Comment by mrousavy** [comment]
😂

**Comment by mrousavy** [comment]
@marcosrjjunior AVFoundationErrorDomain code -11819 means that the daemon has crashed, I think you need to file this bug using Apple's Feedback Assistant. 

I don't think this is related to VisionCamera at all, but it doesn't hurt to also tell them that you use VisionCamera (provide a link to the github repo)

**Comment by marcosrjjunior** [comment]
@mrousavy just an update on this issue.

we were able to workaround that by using [react-native-camera](https://github.com/react-native-camera/react-native-camera), with the option `autoFocus="off"`.

The issue was solved just by disabling that autoFocus prop.
To improve the experience a lit bit, we are using the `autoFocusPointOfInterest` with a "slider" to adjust the focus.

I couldn't find a way to force to disable the auto focus on the current vision camera. Thoughts?

--
UPDATE:
- After further investigations, disabling these options seems to fix the issue.
```swift
self.captureDevice.automaticallyAdjustsFaceDrivenAutoFocusEnabled = NO;
self.captureDevice.faceDrivenAutoFocusEnabled = NO;
```

is there a way to override somewhere [in the camera session](https://github.com/mrousavy/react-native-vision-camera/blob/00caff0288e278e1076cb5b0708bb1ba7f436f07/package/ios/Core/CameraSession%2BConfiguration.swift#L276) without a manual patch?
